### PR TITLE
README updates

### PR DESCRIPTION
**Pull request by hzsweers**
This feels a bit more to the point in the beginning, and not having an example of the generated code in the README felt like a missing piece.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,7 +42,29 @@ class SwitchStencil : ViewStencil(
...
+    return getVisibility() == View.GONE;
+  }
+  
+  public boolean isInvisible() {
+    return getVisibility() == View.INVISIBLE;
+  }
+}
+```
+
+This may look like a lot of boilerplate for simple helpers, but it scales quite well when you want to have these methods on _all_ your base views.
```
Comment (jbarr21) [inline README.md]: maybe we should linkify "all your base views" to the generated views in the sample app? to complete the trait -> stencil -> single view -> all views that Artist is intended for?

Comment (hzsweers) [reply README.md]: I don't think so. You're trying to give a concise gist, and last thing you want to do is take them out of this context and make them look somewhere else for that detail.

**Comment by AttwellBrian** [comment]
In androiddev you mentioned a real world use case: analytics.

Seems like it would be helpful to mention that here.

**Comment by hzsweers** [comment]
True, but I was trying to keep it concise. Analytics are somewhat non-trivial (assumes rx knowledge + a bit nitty gritty)

**Pull request merged**